src/exercise003/Exercise3.java

```diff
@@ -0,0 +1,16 @@
+package exercise003;
+
+/**
+ * Write a Java program to divide two numbers and print on the screen.
+ * Test Data :
+ * 50/3
+ * Expected Output :
+ * 16
+ */
+public class Exercise3 {
+    public static void main(String[] args) {
+        int number1 =50, number2 = 3;
+        double result= number1/number2;
```

Comment:
Tutaj ostrożnie. Takie dzielenie zaokrągli do 16. Jeśli oczekujesz double jako wynik dzielenia to bym się spodziewał, że będziesz chciał pełną dokładność. Albo dać: int result = number1/number2, albo double result= number1/(double)number2

Reply:
Faktycznie jest utrata precyzji. Zrobię castowanie mianownika do double.